bot/extensions/challenges/events.py

```diff
@@ -45,9 +42,8 @@ async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
         if payload.emoji != self.challenge_reaction:
             return
 
-        member = self.bot.guild.get_member(payload.user_id)
-        if member is None or member.bot:
-            return
+        if payload.user_id == self.bot.user.id:
+        return
```

Comment:
You forgot to indent this line.

Reply:
Thank you so much for your review! Let me know if that fixes the issue!